translations/de.ts

```diff
@@ -1431,17 +1435,17 @@ will be sent to them when they appear online to you.</source>
     <message>
         <location filename="../src/widget/groupwidget.cpp" line="92"/>
         <source>%1 users in chat</source>
-        <translation>%1 Kontakte im Chat</translation>
+        <translation>%1 Benutzer im Chat</translation>
     </message>
     <message>
         <location filename="../src/widget/groupwidget.cpp" line="94"/>
         <source>0 users in chat</source>
-        <translation>Keine Kontakte im Chat</translation>
+        <translation>Keine Benutzer im Chat</translation>
```

Comment:
Warum? Kontakte habe ich in meiner Kontaktliste. Fremde im Gruppenchat sind nicht meine Kontakte.